packages/api/AlvTime.Business/Utils/DateFormatter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+namespace AlvTime.Business.Utils
+{
+    public static class DateFormatter
+    {
+        public static string ToDateOnly(this DateTime date)
+        {
+            return date.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);
```

Comment:
Nice! Is this formatting only to be used in the web api? If so, it should might be encapsuled there